ee/fleetctl/updates_test.go

```diff
@@ -152,11 +151,18 @@ func getRoots(t *testing.T, tmpDir string) string {
 	out, err := ioutil.ReadAll(r)
 	require.NoError(t, err)
 
-	// Check output
-	var keys []data.PublicKey
+	// Check output contains the root.json
+	var keys map[string]interface{}
 	require.NoError(t, json.Unmarshal(out, &keys))
-	assert.Greater(t, len(keys[0].IDs()), 0)
-	assert.Equal(t, "ed25519", keys[0].Type)
+	signed_ := keys["signed"]
+	require.NotNil(t, signed_)
+	signed, ok := signed_.(map[string]interface{})
+	require.True(t, ok)
+	keys_ := signed["keys"]
+	require.NotNil(t, keys_)
+	require.NotEmpty(t, signed["keys"])
+	_, ok = keys_.(map[string]interface{})
+	require.True(t, ok)
```

Comment:
Should we check that it has key data like we were doing before?

Reply:
Yes, sorry, I was lazy. Pushed.